pkg/secrethub/credentials/readers.go

```diff
@@ -0,0 +1,83 @@
+package credentials
```

Comment:
Is the idea that the CLI will implement a reader that will prompt?
This can be used as passphrase reader for `UseKey`.

Reply:
Oui oui. That's correct.